ci-operator/config/konflux-ci/release-service-catalog/konflux-ci-release-service-catalog-development.yaml

```diff
@@ -23,7 +23,7 @@ tests:
     cloud: aws
     labels:
       region: us-west-2
```

Comment:
Do we need move it to "us-east-2" if  no more test will use "us-west-2" cluster pool?